gui/mozregui/build_runner.py

```diff
@@ -16,9 +16,8 @@ class GuiBuildDownloadManager(QObject, BuildDownloadManager):
     download_finished = Signal(object, str)
 
     def __init__(self, destdir, persist_limit, **kwargs):
-        QObject.__init__(self)
-        BuildDownloadManager.__init__(
-            self, destdir, session=get_http_session(), persist_limit=persist_limit, **kwargs
+        super().__init__(
+            destdir=destdir, session=get_http_session(), persist_limit=persist_limit, **kwargs
         )
```

Comment:
Referencing this comment here since it disappeared: https://github.com/mozilla/mozregression/pull/1231#pullrequestreview-1418923847